examples/bridge-app/esp32/.gitignore

```diff
@@ -0,0 +1,2 @@
+/sdkconfig
+/sdkconfig.old
```

Comment:
Maybe add "build" too?

Reply:
Added.

I was unsure  about this since I am trying to promote a narative of 'use build_examples.py' to build and then everything goes to out. Agree though we are not there yet and readme instructions likely will advise on creating a build dir.